doc/news/changes/minor/20170104Jean-PaulPelteret

```diff
@@ -0,0 +1,5 @@
+New: LinearOperator now derive from an arbitrary Payload class. This allows
+their functionality to be extended to support algebra class other than
+deal.II's native classes.
```

Comment:
Well, in a sense all of these classes are deal.II classes. Maybe say "... such as the wrappers to PETSc or Trilinos"?